uniffi/src/interface/mod.rs

```diff
@@ -548,6 +572,20 @@ impl Object {
         self.methods.iter().collect()
     }
 
+    pub fn ffi_object_free(&self) -> FFIFunction {
+        FFIFunction {
+            name: format!("ffi_{}_{}_object_free", self.namespace, self.name),
```

Comment:
Ah yes, I see the trick here with needing more of the global context for the namespace. This seems OK I think, thanks for digging in.